app/mailers/notifier.rb

```diff
@@ -50,8 +50,9 @@ def order_receipt(args)
 
   def review_orders(args)
```

Comment:
Would it be worth switching to using keyword arguments? `user_id` and `account_ids` are probably required; not sure about facility.